stargazer.js

```diff
@@ -0,0 +1,67 @@
+import Alt from './'
+import { createStore, datasource } from './utils/decorators'
+import axios from 'axios'
+
+const alt = new Alt()
+
+const StargazerActions = alt.generateActions(
+  'fetchingUsers',
+  'usersReceived',
+  'failed'
+)
+
+const StargazerSource = {
```

Comment:
hrm, it would be nice if this was sorta like a class where we can provide the default implementation of error and success which can be overridden by the child to do additional work if needed. I'm thinking that for all errors, it should display a banner on the page of the message but we don't want to always add that action for every type of data fetch